package.py

```diff
@@ -0,0 +1,27 @@
+name = "PIK_maya_loader"
+
+version = "0.0.1"
```

Comment:
The version must start at `0.1.0`, if you want more details about versioning: https://semver.org/lang/fr/

Here you set the version to a fix but it's new feature (not breaking compatibility), so the minor is more appropriate.